### PR TITLE
Action in SIPNotifierClient is called on incoming notification

### PR DESCRIPTION
**Pull request by ilogixxDE**

### CODE REVIEW

File: src/app/SIPUserAgents/SIPNotifierClient.cs [hunk 1]
```diff
@@ -96,6 +96,20 @@ public string CallID
...
+                {
+                    await GotNotificationRequest(lep, rep, req);
+                }
+                else
+                {
+                    SIPResponse notSupported = SIPResponse.GetResponse(req, SIPResponseStatusCodesEnum.MethodNotAllowed, null);
+                    await sipTransport.SendResponseAsync(notSupported);
+                }
+            };
+
```
Comment (sipsorcery) [inline src/app/SIPUserAgents/SIPNotifierClient.cs:112]: This block isn't necessary. https://github.com/sipsorcery-org/sipsorcery/blob/b9103ea2a2a3321f5cd315dd939c7ef03c7b6470/src/app/SIPUserAgents/SIPNotifierClient.cs#L105 was added this morning to do the same thing.

**Review by sipsorcery — COMMENTED** [review]
I think you might need to rebase your fork. The null checks on the event handlers have already been added from your last PR.

File: src/app/SIPUserAgents/SIPNotifierClient.cs (hunk 1 above)
Comment (ilogixxDE) [reply src/app/SIPUserAgents/SIPNotifierClient.cs:112]: In your change, GotNotificationRequest is called for all Methods, but it should only be called for Notification.

Comment (sipsorcery) [reply src/app/SIPUserAgents/SIPNotifierClient.cs:112]: It's no different to the lambda method. In both cases the `SIPTransportRequestReceived` event has to be caught and the SIP request method checked. It's the first check done in `GetNotificationResult`.

https://github.com/sipsorcery-org/sipsorcery/blob/b9103ea2a2a3321f5cd315dd939c7ef03c7b6470/src/app/SIPUserAgents/SIPNotifierClient.cs#L143

Comment (ilogixxDE) [reply src/app/SIPUserAgents/SIPNotifierClient.cs:112]: So, my PR can be deleted as things are done

**Pull request closed without merging**